sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/TestMethodShouldContainAssertion.cs

```diff
@@ -44,7 +44,11 @@ public sealed class TestMethodShouldContainAssertion : SonarDiagnosticAnalyzer
         private static readonly ISet<KnownType> TrackedTestAttributes = ImmutableHashSet.Create(
             KnownType.Microsoft_VisualStudio_TestTools_UnitTesting_TestMethodAttribute,
             KnownType.NUnit_Framework_TestAttribute,
-            KnownType.Xunit_FactAttribute);
+            KnownType.NUnit_Framework_TestCaseAttribute,
+            KnownType.NUnit_Framework_TestCaseSourceAttribute,
+            KnownType.NUnit_Framework_TheoryAttribute,
+            KnownType.Xunit_FactAttribute,
```

Comment:
Are there tests for the _XUnit FactAttribute_?

Reply:
Fact was there before so I assume there was.